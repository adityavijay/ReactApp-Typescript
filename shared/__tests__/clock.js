import React from 'react';
//import Clock from './components/clock.js';
import renderer from 'react-test-renderer';

const Link = ()=><a  id="link" href="#"> Aditya<span>Hello</span> </a>;


describe(" Clock is here", ()=>{
    it("should do tick",()=>{
        expect(2).toBe(2);
    })
})

const tree = renderer.create(<Link/>).toJSON();
  expect(tree).toMatchSnapshot();