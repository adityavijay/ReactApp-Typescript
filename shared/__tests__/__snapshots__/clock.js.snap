// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<a
  href="#"
  id="link"
>
  <span>
    Hello
  </span>
   Aditya
</a>
`;
